astropy/cosmology/_src/utils.py

```diff
@@ -105,7 +108,7 @@ def aszarr(
 
 
 def deprecated_keywords(
-    *kws: str, since: str | float | tuple[str | float, ...]
```

Comment:
oh wait, did I write this ? 😅 
Any way, it's still private, so we can still get away with *removing* undue flexibility now.